### DISCUSSION TITLE
pt/integrations/openvino/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#11697] [Docs]
# pt/integrations/openvino/

Descobre o poder da implementação do teu modelo Ultralytics YOLOv8  utilizando o formato OpenVINO para uma aceleração até 10x superior a PyTorch.

https://docs.ultralytics.com/pt/integrations/openvino/

**Comment by renato-rodrig** [comment]
hello everything is fine? I have a question about the results exposed in the article above. Is the time shown in the graphs in relation to the inference time that the model takes to return the detections or is it the time for image reading, image pre-processing and model inference? I ask this because on my computer (intel core i7 12 generation CPU) the yolov8n model ran with an inference time of 40ms and the gain from using OpenVINO was not that great.

**Reply by pderrenger** [reply]
@renato-rodrig hello! 👋 Regarding your question about the times shown in the graphs, they typically refer to the total inference time, which includes the actual model inference as well as the time taken for pre-processing the input data (like image reading and resizing). 

The difference you observed with OpenVINO on your Intel Core i7 may vary due to several factors such as the specific configuration of your system, the type of tasks being run, and the optimizations that the YOLOv8n model can leverage on that platform. Generally, OpenVINO tends to show more significant performance gains on integrated Intel GPUs and some specific CPU configurations.

If you haven't seen a major performance improvement, you might want to look into further tuning your OpenVINO deployment or explore using other optimization techniques provided by OpenVINO. Additionally, ensure all drivers and software components are up to date for optimal performance.

Here’s a quick example of ensuring you're using the model in the most optimized manner:

```python
from ultralytics import YOLO

# Load YOLOv8n PyTorch model
model = YOLO('yolov8n.pt')

# Export the model to OpenVINO
model.export(format='openvino')  # Creates 'yolov8n_openvino_model/'

# Load the exported OpenVINO model
ov_model = YOLO('yolov8n_openvino_model/')

# Run inference to see if performance improves
results = ov_model('https://ultralytics.com/images/bus.jpg')
```

Feel free to experiment with different configurations and settings! 😊
